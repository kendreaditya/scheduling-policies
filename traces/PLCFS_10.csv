1,0,0
2,0,0
3,0,0
4,0,0
5,0,50
6,0,100
7,0,50
8,0,100
9,0,50
10,0,100
11,0,50
12,0,100
13,0,0
14,0,0
15,0,50
16,0,100
17,0,100
18,0,50
19,0,100
20,0,50
