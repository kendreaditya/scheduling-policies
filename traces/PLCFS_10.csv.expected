1,0
2,0
3,0
4,0
5,1050
6,1000
7,900
8,850
9,750
10,700
11,600
12,550
13,0
14,0
15,450
16,400
17,300
18,200
19,150
20,50
